src/structures/Sticker.js

```diff
@@ -9,10 +9,6 @@ const SnowflakeUtil = require('../util/SnowflakeUtil');
  * @extends {Base}
  */
 class Sticker extends Base {
-  /**
-   * @param {Client} client The instantiating client
-   * @param {APISticker | APIStickerItem} sticker The data for the sticker
```

Comment:
This was the last instance of `APIStickerItem`. Should we remove this too?
https://github.com/discordjs/discord.js/blob/395186da7241b1ea704e4d5e0c6df6fd4655eb5c/src/structures/Sticker.js#L238-L241